examples/tutorials/advanced/Federated Recurrent Neural Network.ipynb

```diff
@@ -0,0 +1,985 @@
+{
```

Comment:
<p>can you add a space before and after all the "=" at variables declaration?</p>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Done!</p>